library/src/main/java/com/google/maps/android/clustering/view/DefaultClusterRenderer.java

```diff
@@ -842,6 +947,7 @@ private void perform(MarkerModifier markerModifier) {
                         } else {
                             markerOptions.position(item.getPosition());
                         }
+                        // TODO (?) - Refactor set marker text into onBeforeClusterItemRendered()
```

Comment:
@arriolac Thoughts on refactoring the code below to `onBeforeClusterItemRendered()`? This would better match the implementation of `onBeforeClusterRendered()` for clusters and also allow applications extending `DefaultClusterRenderer` to inherit this default behavior along with a custom implementation via a call to `super.onBeforeClusterItemRendered()`. However, moving this code would change the behavior of the library for apps already extending `DefaultClusterRenderer` that aren't calling `super.onBeforeClusterItemRendered()` - the marker title and snippet would no longer be updated.

Reply:
I think it only makes sense to refactor if the default implementation of updating title/snippet on the `MarkerOptions` is not desired. Seems that we'd always want that though. Can you think of a scenario where that might not be the case?

Reply:
I think this warrants it's own issue, especially given that the potential refactor would alter behavior - I'm going to remove the TODO in this PR and we can discuss in that thread. In short, yes - we actually override this behavior in the demo `CustomMarkerClusteringDemoActivity`.